docs/content/en/docs/migrate/evolution-klt/_index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Evolution of KLT
+description: Understand the paradigm of KLT, how it evolved from Keptn v1, and whether you should migrate to KLT
+weight: 5
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+The Keptn products provide tool-agnostic ways
+to manage and observe software deployment.
+Keptn connects to the tools you choose
+through an open event standard
```

Comment:
Keptn v1 does. KLT doesn't use the NATS eventing at all. KLT _emits_ events to Kubernetes - but that's opinionated to Kubernetes.

Reply:
Added as TODO to source